src/CSSBattle/6_MissingSlice/ZhangZhikai.html

```diff
@@ -0,0 +1,42 @@
+
+<div class="shape top">
+  <div class="shape">
+  </div>
+  <div class="shape bottom">
+  </div>
+  
+  </div>
+  
+  
```

Comment:
Consider removing unnecessary line break and indentations

Reply:
50ed92b92dcb6d7fac9a9b6965067d8e1abde559